scene/gui/rich_text_label.cpp

```diff
@@ -6763,6 +6771,7 @@ void RichTextLabel::_generate_context_menu() {
 	add_child(menu, false, INTERNAL_MODE_FRONT);
 	menu->connect(SceneStringName(id_pressed), callable_mp(this, &RichTextLabel::menu_option));
 
+	menu->add_item(ETR("Clear"), MENU_CLEAR);
```

Comment:
Having a disabled option in every RTL just for editor log is not the best way. There's `get_menu()` in RTL, so `EditorLog` can add an item to its menu directly.